pkg/cmd/start/main.go

```diff
@@ -34,6 +34,7 @@ func AddFlags(cmd *cobra.Command) {
 	cmd.Flags().String("es-provision", "auto", "Whether to auto-provision an Elasticsearch cluster for suitable Jaeger instances. Possible values: 'yes', 'no', 'auto'. When set to 'auto' and the API name 'logging.openshift.io' is available, auto-provisioning is enabled.")
 	cmd.Flags().String("kafka-provision", "auto", "Whether to auto-provision a Kafka cluster for suitable Jaeger instances. Possible values: 'yes', 'no', 'auto'. When set to 'auto' and the API name 'kafka.strimzi.io' is available, auto-provisioning is enabled.")
 	cmd.Flags().Bool("kafka-provisioning-minimal", false, "(unsupported) Whether to provision Kafka clusters with minimal requirements, suitable for demos and tests.")
+	cmd.Flags().Bool("collector-otlp-enabled", true, "Expose otlp ports on the collector.")
```

Comment:
I would say the flag is not needed, and rather the operator should check if the OTLP collector flag is defined in the CR options and disable it if the value is false. 

We also don't expose the Zipkin enable flag on the operator.